plonky2/src/plonk/circuit_builder.rs

```diff
@@ -524,6 +524,13 @@ impl<F: RichField + Extendable<D>, const D: usize> CircuitBuilder<F, D> {
             .push(CopyConstraint::new((x, y), self.context_log.open_stack()));
     }
 
+    /// Enforces that two [`Target`] arrays are equal.
```

Comment:
I'm not convinced equal here is especially clear.

Reply:
should I precise `underlying values` perhaps?

Reply:
Yeah that's more clear